wikidev/wikidev/views.py

```diff
@@ -28,68 +34,82 @@ def get_user(self):
         data = jwt.decode(token.replace('Bearer ', ''), key='87654321')
         user, created = User.objects.get_or_create(id=data['sub'],
                                                    username=str(data['sub']))
-       # user, created = User.objects.get_or_create(id=231243,
-       #                                            username='Mircea')
         return user
 
-    def to_response(self, menu_or_form):
-        response = onem.Response(
-                menu_or_form, self.request.headers['X-Onem-Correlation-Id']
-        )
-        return HttpResponse(response.as_json(), content_type='application/json')
+    def to_response(self, content):
+        response = Response(content=content)
+        response.correlation_id = self.request.headers['X-Onem-Correlation-Id']
+
+        return HttpResponse(response.json(), content_type='application/json')
 
 
 class HomeView(View):
     http_method_names = ['get']
 
     def get(self, request):
-        #user = self.get_user()
-
-        body = [
-            onem.menus.MenuItem('Search', url=reverse('search_wizard')),
-            onem.menus.MenuItem('Random', url=reverse('random')),
-            onem.menus.MenuItem('Language', url=reverse('language'))
+        #user = self.get_user()  # TODO
+
+        menu_items = [
+            MenuItem(type=MenuItemType.option,
+                     description='Search',
+                     method='GET',
```

Comment:
Not sure but the method should default to `GET` at Nautilus level. It can also default to `GET` at the SDK level I think, because most probably the majority of menus will use GET.

Reply:
Indeed, the majority of menus will use GET so I think defaulting to GET in the SDK is a good idea.